bin/pycbc_inspiral

```diff
@@ -25,6 +25,7 @@ from pycbc.types import TimeSeries, FrequencySeries, zeros, float32, complex64
 import pycbc.fft.fftw, pycbc.version
 import pycbc.opt
 import pycbc.weave
+import pycbc.injcutter
```

Comment:
Surely --injection-chirp-time-threshold is no more clear than --injcutter-chirp-time-threshold. There is no --injection-skip-templates (or similar) option, they are all sub-options.

I can change threshold to window.